.github/workflows/ci.yml

```diff
@@ -75,8 +75,8 @@ jobs:
         sudo apt-get install gcovr ${{ matrix.platform.name == 'Linux Clang' && 'llvm-$CLANG_VERSION' || '' }}
 
     - name: Install macOS Tools
-      if: matrix.type.name == 'Debug' && runner.os == 'macOS'
```

Comment:
I guess there's no noticeable time penalty in installing gcovr even when we don't use it in release mode?

Reply:
That's what I was banking on. I tried installing it locally and it only took 8 seconds so I thought we could get away with always doing that.